### ISSUE TITLE
mediaDevices.enumerateDevices returns different deviceIds on every reload

### ISSUE THREAD

**Issue by codedrift** [electron/electron#22144]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * Tested with 8.0.0 and v9.0.0-beta.1
* **Operating System:**
  * macOS 10.15.3 (19D76)
* **Last Known Working Electron version:**
  * 7.1.12

### Expected Behavior

`navigator.mediaDevices.enumerateDevices()` should return a list of MediaDevices with a `label` and a `deviceId` property. The `deviceId` property should stay the same across reloads and restarts.

### Actual Behavior
The `deviceId` property changes with every reload/restart.

### To Reproduce
Simply execute `navigator.mediaDevices.enumerateDevices()` in the renderer process and compare the ids of the media devices to a different execution.

See this fork for an example:
https://github.com/codedrift/electron-quick-start

```sh
$ git clone https://github.com/codedrift/electron-quick-start -b media-device-ids
$ npm install
$ npm start || electron .
```

### Additional Information
It seems to work fine in a regular Chrome 80 browser.

**Comment by erickzhao** [comment]
> The deviceId property should stay the same across reloads and restarts.

Note: it seems like `deviceId` also changed between app restarts for Electron <=7.

**Comment by erickzhao** [comment]
Seems to fail after this Chromium roll: https://github.com/electron/electron/pull/20649

**Comment by lyswhut** [comment]
Same problem.

- Electron Version: 8.0.1
- Operating System: Windows 10

**Comment by koenoe** [comment]
Can I expect this fix in 8.0.4 and 9.0.0-beta-6?
